### PR TITLE
Add new dialog for login button

### PR DESCRIPTION
**Pull request by MartynasStrazdas**
Added new dialog without backEnd for login button.

### CODE REVIEW

File: karma/Components/Dialogs/DialogLogin.razor [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        </MudText>
+    </TitleContent>
+
+    <DialogContent>
+        <MudTextField @bind-Value="@_username"
+                      Label="Username"
+                      Required="true"
+                      RequiredError="Username is required" />
+        <MudTextField @bind-Value="@_password"
+                      T="string" Label="Pasword"
```
Comment (JustasGasparaitis) [inline karma/Components/Dialogs/DialogLogin.razor:16]: Typo

File: karma/Components/Dialogs/DialogLogin.razor [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+                      Variant="Variant.Text"
+                      Lines="5"
+                      Required="true"
+                      RequiredError="Password is required" />
```
Comment (JustasGasparaitis) [inline karma/Components/Dialogs/DialogLogin.razor:20]: Check out password type text fields: 
https://mudblazor.com/components/textfield#form-props

File: karma/Shared/NavMenu.razor [hunk 3]
```diff
@@ -34,5 +38,14 @@
...
-    //
+    private async Task OpenDialog()
+    {
+        var dialog = _dialogService.Show<DialogLogin>("Login");
+        var result = await dialog.Result;
+
+        if (!result.Cancelled)
+        {
+
+        }
```
Comment (JustasGasparaitis) [inline karma/Shared/NavMenu.razor:49]: Why is this just an empty if statement?

Comment (MartynasStrazdas) [reply karma/Shared/NavMenu.razor:49]: I left it for the future back end

File: karma/Components/Dialogs/DialogLogin.razor (hunk 1 above)
Comment (MartynasStrazdas) [reply karma/Components/Dialogs/DialogLogin.razor:16]: Ups, will fix that

**Review by JustasGasparaitis — APPROVED** [review]

**Pull request merged**